org.plomgrading.PlomClient.json

```diff
@@ -0,0 +1,46 @@
+{
+    "app-id": "org.plomgrading.PlomClient",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "PlomClient",
+    "finish-args": [
+        "--socket=x11",
+        "--share=network"
+    ],
+    "modules": [
+        {
+            "name": "hello",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D PlomClient.workaround /app/bin/PlomClient"
+            ],
+            "sources": [
+                {
```

Comment:
You can just add this file as a second source to the `Plom` module and install it in `build-commands`